blueocean-commons/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.jenkins.blueocean</groupId>
     <artifactId>blueocean-parent</artifactId>
-    <version>1.0.0-b15-SNAPSHOT</version>
+    <version>1.0.0-beta-15-SNAPSHOT</version>
```

Comment:
how come this is change to be the experimental update centre? I guess a temporary thing?